src/Core/TestSuite/ContainerStubTrait.php

```diff
@@ -81,7 +81,9 @@ public function configApplication(string $class, ?array $constructorArgs): void
      */
     protected function createApp(): HttpApplicationInterface|ConsoleApplicationInterface
     {
-        Router::resetRoutes();
+        if (class_exists('Cake\Routing\Router')) {
```

Comment:
```suggestion
        if (class_exists(Router::class)) {
```

The class-string expansion doesn't need the class to be present.